.travis.yml

```diff
@@ -9,6 +12,8 @@ before_install:
     - sudo make install
     - sudo ldconfig /usr/local/lib
 
+before_script: "echo \"db.test.insert({ foo: 42 }); db.getSiblingDB('sample').movies.insert({ title: 'Jaws' });\" | mongo"
```

Comment:
What's the reasoning behind putting { foo: 42 } in the database before running the tests? Right now, the wire protocol tests assume that the database is empty initially.